src/panel/ServicePanelMobile.jsx

```diff
@@ -0,0 +1,104 @@
+/* global _ */
+import React from 'react';
+import { Fragment } from 'react';
+import Panel from 'src/components/ui/Panel';
+import CategoryService from 'src/adapters/category_service';
+import CategoryList from 'src/components/CategoryList';
+import nconf from '@qwant/nconf-getter';
+
+const directionConf = nconf.get().direction;
+
+class ServicePanelMobile extends React.Component {
+  render() {
+    return <Panel
+      resizable
+      title=""
+      minimizedTitle={_('Unfold to see quick actions', 'service panel')}
+      className="service_panel"
+      white
+    >
+      <div className="service_panel__actions">
+
+        <h3>{_('Itineraries', 'service panel')}</h3>
+
+        {
+          nconf.get().direction.enabled &&
+          <Fragment>
+            <button
+              onClick={() => { window.app.navigateTo('/routes/?mode=driving'); }}
+              className="service_panel__action service_panel__item__direction">
+              <div className="service_panel__action__icon">
+                <span className="icon-drive"/>
+              </div>
+              <div className="service_panel__action__title">{_('by car', 'service panel')}</div>
+            </button>
+            {
+              directionConf.publicTransport
+              && directionConf.publicTransport.enabled
+              && <button
+                onClick={() => {
+                  window.app.navigateTo('/routes/?mode=publicTransport');
+                }}
+                className="service_panel__action service_panel__item__direction">
+                <div className="service_panel__action__icon">
+                  <span className="icon-public-transport"/>
+                </div>
+                <div className="service_panel__action__title">
+                  {_('transport', 'service panel')}
+                </div>
+              </button>
+            }
+            <button
+              onClick={() => { window.app.navigateTo('/routes/?mode=walking'); }}
+              className="service_panel__action service_panel__item__direction">
+              <div className="service_panel__action__icon">
+                <span className="icon-foot"/>
+              </div>
+              <div className="service_panel__action__title">{_('on foot', 'service panel')}</div>
+            </button>
+            <button
+              onClick={() => { window.app.navigateTo('/routes/?mode=cycling'); }}
+              className="service_panel__action service_panel__item__direction">
+              <div className="service_panel__action__icon">
+                <span className="icon-bike"/>
+              </div>
+              <div className="service_panel__action__title">{_('on bike', 'service panel')}</div>
```

Comment:
"by bike" would probably be more correct ?

Reply:
yes. I googled the proverbs for each vehicle and thought the answer was "on" but "by" is indeed better